plotly/tests/test_core/test_spectacle_presentation/test_spectacle_presentation.py

```diff
@@ -0,0 +1,207 @@
+"""
+test_spectacle_presentation:
+==========
+
+A module intended for use with Nose.
+
+"""
+from __future__ import absolute_import
+
+from unittest import TestCase
+from plotly.exceptions import PlotlyError
+import plotly.presentation_objs as pres
+
+
+class TestPresentation(TestCase):
+
+    def test_invalid_style(self):
+        markdown_string = """
+        # one slide
+        """
+
+        message = 'Your presentation style must be moods or martik.'
+
+        self.assertRaisesRegexp(PlotlyError, message, pres.Presentation,
+                                markdown_string, style='foo')
+
+    def test_open_code_block(self):
+        markdown_string = """
+        # one slide
+
+        ```python
+        x = 2 + 2
+        print x
+        """
+
+        message = (
+            "If you are putting a block of code into your markdown "
+            "presentation, make sure your denote the start and end "
+            "of the code environment with the '```' characters. For "
+            "example, your markdown string would include something "
+            "like:\n\n```python\nx = 2\ny = 1\nprint x\n```\n\n"
+            "Notice how the language that you want the code to be "
+            "displayed in is immediately to the right of first "
+            "entering '```', i.e. '```python'."
+        )
```

Comment:
instead of hardcoding these messages, could you import them from the original file? That way, if we wanted to change these messages, we would only have to do it from one place